convert2rhel/toolopts.py

```diff
@@ -19,7 +19,9 @@
 import logging
 import optparse
 import os
+import re
 import sys
+import urllib.parse
```

Comment:
`No name 'parse' in module 'urllib'` on CentOS Linux 7.

Reply:
The CI is failing right now with the same errror

https://download.copr.fedorainfracloud.org/results/@oamg/convert2rhel/epel-7-x86_64/04726137-convert2rhel/builder-live.log.gz
https://download.copr.fedorainfracloud.org/results/@oamg/convert2rhel/epel-6-x86_64/04726137-convert2rhel/builder-live.log.gz

Reply:
Yeah, urllib.parse was introduced in python3.  I switched the imports to use `from six.moves import urllib` which should resolve this.